downstairs/src/lib.rs

```diff
@@ -3350,6 +3351,72 @@ pub async fn start_downstairs(
     Ok(join_handle)
 }
 
+/// Clone the extent files in a region from another running downstairs.
+///
+/// Use the reconcile/repair extent methods to copy another downstairs.
+/// The source downstairs must have the same RegionDefinition as we do,
+/// and both downstairs must be running in read only mode.
+pub async fn start_clone(
+    d: Arc<Mutex<Downstairs>>,
+    source: SocketAddr,
+) -> Result<()> {
+    let info = crucible_common::BuildInfo::default();
+    let log = d.lock().await.log.new(o!("task" => "main".to_string()));
```

Comment:
Why `"main"` instead of `"clone"`?

Reply:
It is the "main" task for the clone command, but I'm not attached to the name really.
I'll change it to clone.

Reply:
Done in [7bef555](https://github.com/oxidecomputer/crucible/commit/7bef55560bd1a345855ca77b6c296616ee66e0fe)